### PR TITLE
Fix menu

### PR DESCRIPTION
**Pull request by webbertakken**
#### Changes

- Add category configurations to capitalise sidebar folder items.
- Upgrade to canary version to get the bugfix from https://github.com/facebook/docusaurus/issues/7348

image

#### Checklist

- [x] Read the contribution [guide](../blob/main/CONTRIBUTING.md) and accept the [code](../blob/main/CODE_OF_CONDUCT.md) of conduct
- [x] Readme (updated or not needed)
- [x] Tests (added, updated or not needed)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,10 +21,10 @@
     "typecheck": "tsc"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-beta.20",
-    "@docusaurus/module-type-aliases": "^2.0.0-beta.20",
-    "@docusaurus/preset-classic": "^2.0.0-beta.20",
-    "@docusaurus/theme-common": "^2.0.0-beta.20",
+    "@docusaurus/core": "^0.0.0-5035",
```
Comment (GabLeRoux) [inline package.json:24]: is this normal?

**Review by GabLeRoux — APPROVED** [review]
There seem to be some changes about markdown formatting mixed up in the changes, but overall I'm ok with the results.

I left a comment concerning changes in `package.json`, but the changes about categories does fix the problem so I'm good with that. Nice work :D

edit: answer in description, lgtm!

File: package.json (hunk 1 above)
Comment (webbertakken) [reply package.json:24]: See description.

Comment (GabLeRoux) [reply package.json:24]: on mobile here, thanks for pointing out, missed that :)

Comment (webbertakken) [reply package.json:24]: As for the formatting, that's just the pre-commit hook. Seems like they were skipped before.

**Pull request merged**